panacea/grpc_client.go

```diff
@@ -52,7 +55,29 @@ type GrpcClient struct {
 
 func NewGrpcClient(conf *config.Config) (GrpcClientI, error) {
 	log.Infof("dialing to Panacea gRPC endpoint: %s", conf.Panacea.GRPCAddr)
-	conn, err := grpc.Dial(conf.Panacea.GRPCAddr, grpc.WithTransportCredentials(insecure.NewCredentials()))
+
+	var conn *grpc.ClientConn
+	var err error
+
+	u, err := url.Parse(conf.Panacea.GRPCAddr)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse grpc addr: %w", err)
+	}
+
+	prefixLen := len(u.Scheme + "://")
+	addrBody := conf.Panacea.GRPCAddr[prefixLen:]
```

Comment:
@youngjoon-lee @inchori @hansol-medi 
What do you think of this code suggestion?
```suggestion
	addrBody := u.Host + u.Path
```

Reply:
It seems to be better.

Reply:
I hope we don't use the addr that is received from the user as it is. We can't be sure if the `u.Host + u.Path` is enough to communicate with the gRPC server. According to the gRPC server configuration, we may need additional values such as `u.User` or `u.Fragment`. 

So, I think it's better to not re-assemble the URL.

Reply:
fyi, tendermint does this in worse: https://github.com/tendermint/tendermint/blob/51b3f111dceb2ffb9ee0e756283303aba608579c/types/node_info.go#L283
:)

Reply:
Good.
Let's not modify this code. 👍

Reply:
Thanks for comments and reference code!.